### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jfmoron**
Updated the readme to include the note for eLux 6.9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ DigiCert SHA2 Assured ID Code Signing CA:
 
 https://cacerts.digicert.com/DigiCertSHA2AssuredIDCodeSigningCA.crt.pem
 
+### Browser Container Certificates
+
+Elux 6.9 brings certificate management directly in the OS for our browser container. If you need to use custom certificates for reaching https websites in the browser container please ignore the Certificate Configuration below and put the certificate in the /setup/cacerts/browser folder as mentionned in the eLux [Documentation](https://www.unicon-software.com/udocs/en/#admin_guides/scout_enterprise/app_definition/browser/browser_config.htm?Highlight=cacert).
```
Comment (brenthompson) [inline README.md:21]: Elux -> eLux ?
brings -> provides ?
"if you need" -> "if there is a need" ?
mentionned -> mentioned

**Review by brenthompson — CHANGES_REQUESTED** [review]
only the 'mentionned' typo is important, the others changes are just suggestions

File: README.md (hunk 1 above)
Comment (jfmoron) [reply README.md]: fixed thanks

**Review by brenthompson — APPROVED** [review]
looks good

**Pull request merged**